templates/html/navtree.css

```diff
@@ -65,6 +65,9 @@
   padding:0px;
 }
 
+#nav-tree .link {
+}
+
```

Comment:
I think the same can be already be achieved via a descendant selector, e.g. using

```.css
#nav-tree span.label a {
        background: red;
}
```
which will style all `a` tags under `span.label` inside `#nav-tree`.

Reply:
This is indeed the case, it is a little bit less clear (to me) but it works.